activerecord/test/cases/validations/uniqueness_validation_test.rb

```diff
@@ -386,4 +393,13 @@ def test_validate_uniqueness_on_empty_relation
     topic = TopicWithUniqEvent.new
     assert topic.valid?
   end
+
+  def test_validate_uniqueness_on_model_without_primary_key
+    MateyWithUniqValidation.create(pirate_id: 1)
+    matey = MateyWithUniqValidation.create(pirate_id: 2)
+
+    matey.decrement(:pirate_id)
```

Comment:
Do you think this test would be more clear if this and the previous line were written as:

```ruby
matey = MateyWithUniqValidation.new(pirate_id: 1)
```